exes/operations/inc/state_machine_owner.hpp

```diff
@@ -0,0 +1,129 @@
+#pragma once
+#include <memory>
+#include <system_error>
+
+#include <boost/sml.hpp>
+
+#include <tfc/confman.hpp>
+#include <tfc/ipc.hpp>
+#include <tfc/ipc/details/dbus_client_iface.hpp>
+#include <tfc/ipc/details/type_description.hpp>
+#include <tfc/operation_mode/common.hpp>
+#include <tfc/sml_logger.hpp>
+#include <tfc/stx/concepts.hpp>
+#include <tfc/utils/asio_fwd.hpp>
+#include <tfc/utils/pragmas.hpp>
+
+namespace tfc::operation {
+
+namespace asio = boost::asio;
+using new_mode = mode_e;
+using old_mode = mode_e;
+
+namespace detail {
+template <typename owner_t>
+struct state_machine;
+
+struct storage {
+  std::optional<std::chrono::milliseconds> startup_time{};
+  std::optional<std::chrono::milliseconds> stopping_time{};
+  struct glaze {
+    // clang-format off
+    static constexpr auto value{ glz::object(
+        "startup_time", &storage::startup_time, "[ms] Delay to run initial sequences to get the equipment ready.",
```

Comment:
Is [ms] required. The frontend converts and marks this type.

Reply:
not really

Reply:
removed